lib/logstash/filters/mutate.rb

```diff
@@ -4,7 +4,7 @@
 
 # The mutate filter allows you to perform general mutations on fields. You
 # can rename, remove, replace, and modify fields in your events.
-class LogStash::Filters::Mutate < LogStash::Filters::Base
+module LogStash module Filters class Mutate < LogStash::Filters::Base
```

Comment:
This is because Base is not declared in the Filters module as yet.  We can when we modify filters/base to use the same pattern. I was not confident enough to make such sweeping changes to the Logstash gem itself.